wechatpay/src/test/java/com/stripe/android/payments/wechatpay/PaymentIntentFixtures.kt

```diff
@@ -0,0 +1,145 @@
+package com.stripe.android.payments.wechatpay
+
+import com.stripe.android.model.parsers.PaymentIntentJsonParser
+import org.json.JSONObject
+
+object PaymentIntentFixtures {
+    private val PARSER = PaymentIntentJsonParser()
+
+    private val PI_REQUIRES_BLIK_AUTHORIZE_JSON = JSONObject(
```

Comment:
Can we omit the objects we aren't using?

Reply:
This is used in `authenticate should throw exception when stripeIntent is not WeChatPay` test